<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="subTwoButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>600</y>
      <width>389</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать 2 шара</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="firstGroupBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Вывод из Двух корзин</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>391</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="firstGridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="procentAllRedBalls">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Вероятность достать 2 красных шара</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="procentOneBlueOneRed">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Вероятность достать 1 красный и 1 синий шар</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="procentAllBlueBalls">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Вероятность достать 2 синих шара</string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>1311</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="basketsContainer"/>
   </widget>
   <widget class="QDateTimeEdit" name="dateFrom">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>145</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2025</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>145</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2025</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logView">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortLogButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать сортировку</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1329</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
